README.md

```diff
@@ -8,16 +8,165 @@ Local.
 For more information, see [Getting started with Cloudant NoSQL DB](https://www.ng.bluemix.net/docs/services/Cloudant/index.html)
 
 The `loopback-connector-cloudant` module is the Cloudant connector for the Loopback framework.
+<!-- TOC -->
+
+- [loopback-connector-cloudant](#loopback-connector-cloudant)
```

Comment:
@ssh Sorry I am confused what do we put a PR hyperlink here? This is auto generated by TOC.